### PR TITLE
use a PHPUnit bridge version that is compatible with recent PHPUnit releases

### PR DESCRIPTION
**Pull request by xabbuh**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| Documentation   | 
| License         | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "php-http/client-common": "^1.9 || ^2.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^5.3",
+        "symfony/phpunit-bridge": "^5.4.33 || ~6.3.10 || ^6.4.1",
```
Comment (dbu) [inline composer.json]: do we need to allow 6.3.10? 

as this is only require-dev, i'd like to keep the list as short as possible.

**Review by dbu — COMMENTED** [review]
thanks for looking into this.

the phpstan error is our friend from https://github.com/php-http/client-common/pull/234#issuecomment-1835626700 - not related to your changes.

can you maybe check the flex composer plugin error? maybe we just never built this package since composer started having configuration to control plugin execution and we need to allow the flex plugin.

**Comment by KDederichs** [comment]
I fixed the Flex error in my PR, if you merge this I can merge it into mine and it should be good I think

File: composer.json (hunk 1 above)
Comment (xabbuh) [reply composer.json]: in fact, using `^6.4.1` is enough

**Comment by dbu** [comment]
thank you!

merged in #24

**Pull request closed without merging**